compiler/rustc_interface/src/tests.rs

```diff
@@ -615,6 +615,7 @@ fn test_codegen_options_tracking_hash() {
     tracked!(embed_bitcode, false);
     tracked!(force_frame_pointers, FramePointer::Always);
     tracked!(force_unwind_tables, Some(true));
+    tracked!(hint_mostly_unused, false);
```

Comment:
It must be different from the default value of the option.
```suggestion
    tracked!(hint_mostly_unused, true);
```